### ISSUE TITLE
Benchmark training performance

### ISSUE THREAD

**Issue by bspivey** [amdegroot/ssd.pytorch#559]
I am training on the VOC dataset. The code appears to be training on both the 2007 and 2012 data sets: `image_sets=[('2007', 'trainval'), ('2012', 'trainval')]`

I reduced the learning rate to 1e-5 and kept all other parameters the same. After 70000 iterations, half the max iterations, My losses are averaging ~4.9, and when I apply the latest weights to the demo image, the model detects only three people and no bottle or dining table.

The training time required makes evaluating training settings more challenging. Does anyone have any guidance on improved performance with other gamma, lr, or momentum values?[](url)

**Comment by ksmdnl** [comment]
I think that makes sense, since the training would usually take more time with lower learning rate, hence slower convergence.
One of these approaches should improve the performance: normalising the input on the detector head (if not on each layer) or selecting other optimizer such as Adam leads to relatively faster convergence.
